mujoco_warp/_src/solver.py

```diff
@@ -516,15 +515,17 @@ def linesearch_parallel_fused(
   efc_done_in: wp.array(dtype=bool),
   njmax_in: int,
   nacon_in: wp.array(dtype=int),
-  # Data out:
-  efc_cost_candidate_out: wp.array2d(dtype=float),
+  # In:
+  nstep: int,
```

Comment:
this should be named `opt_ls_iterations` no?

Reply:
done